### DISCUSSION TITLE
eMMC life indicator

### DISCUSSION THREAD

**Discussion by AuthorShin** [IceWhaleTech/CasaOS#487] [Ideas]
Add a estimator for eMMC on the Zimaboards to know how far they will last.

**Comment by Hedius** [comment]
How should that work? Tools like smart are not available for emmc storages?

**Reply by AuthorShin** [reply]
It's more of a reminder for @JohnGuan  than a initial idea post since he had a way for it, but [here](https://electronics.stackexchange.com/a/525525) I think is a way for that if eMMC is above 5.1.
